skaffold.yaml

```diff
@@ -17,6 +17,10 @@ build:
             - VERSION
         flags:
           - -v
+        ldflags:
+          - -w
+          - -X github.com/gardener/etcd-druid/internal/version.Version={{.VERSION}}
```

Comment:
We are not explicitly setting the VERSION; it must be passed externally when invoking Skaffold. [Prow e2e test](https://prow.gardener.cloud/view/gs/gardener-prow/pr-logs/pull/gardener_etcd-druid/842/pull-etcd-druid-e2e-kind/1816302820614213632#1:build-log.txt%3A1033-1124) is failing. Since VERSION is not set, it defaults to an empty string, leading to an incorrect configuration:

```
.PHONY: deploy
deploy: $(SKAFFOLD) $(HELM)
	@VERSION=$(VERSION) GIT_SHA=$(GIT_SHA) $(SKAFFOLD) run -m etcd-druid
```

Reply:
Ah you're right. Will fix this, thanks

Reply:
On second thought, `make deploy` is never called from `make ci-e2e-kind`. Skaffold is called from [here](https://github.com/gardener/etcd-druid/blob/c73cbf9c5715ff157694a919ee73d079d668c0aa/hack/e2e-test/run-e2e-test.sh#L98), via `make test-e2e`. So the change needed to be made there. I have pushed the new changes now. PTAL.

Also, I have added some helper Make targets for easily cleaning up go build cache and docker builds for druid.

Reply:
👍